spec/aci.md

```diff
@@ -239,6 +240,7 @@ JSON Schema for the Image Manifest (app image manifest, ACI manifest), conformin
         * **port** (integer, required) port number that will be used; see also **count**. Must be >=1 and <=65535.
         * **count** (integer, optional, defaults to 1 if unset) specifies a range of ports, starting with "port" and ending with "port" + "count" - 1. Must be >=1.
         * **socketActivated** (boolean, optional, defaults to "false" if unsupplied) if set to true, the application expects to be [socket activated](http://www.freedesktop.org/software/systemd/man/sd_listen_fds.html) on these ports. To perform socket activation, the ACE MUST pass file descriptors using the [socket activation protocol](http://www.freedesktop.org/software/systemd/man/sd_listen_fds.html) that are listening on these ports when starting this app. If multiple apps in the same pod are using socket activation then the ACE must match the sockets to the correct apps using getsockopt() and getsockname().
+    * **supportNotify** (boolean, optional, defaults to "false" if unset) if set to true, the application should use [sd_notify()](https://www.freedesktop.org/software/systemd/man/sd_notify.html) to signal when it is ready. Setting this parameter to true for an application that does not send a ready signal with `sd_notify()` may result in misbehave.
```

Comment:
> What MUST executors do if an application sets this parameter to true but does not send a ready signal?
Also, what SHOULD executors do if an application sets this parameter to true but the executor does not support sd_notify?

In both case ma answer is that this should not happen, because setting to true `supportNotify` means that the app will be responsible for the notification. For example, in systemd, if the init system do not get the ready message after a timeout will fill the app.

I could write something like: "Setting this parameter to true for an application that does not send a ready signal with `sd_notify()` may result in starvation or termination of the app. When setting to true the app will be responsible to provide the minimum notifications to the init system"